### PR TITLE
Improvement: Benchmarking kit with SambaStudio URL

### PR DESCRIPTION
**Pull request by snova-rodrigom**
- Changed code to include tqdm for CLI runs
- Added changes to make sambanova client work with just URL and api key in kit
- Tested functionalities for prod and non-prod scenarios
- Ran ruff and mypy

### CODE REVIEW

File: benchmarking/streamlit/app.py [hunk 1]
```diff
@@ -92,11 +91,7 @@ def main() -> None:
                 st.session_state.llm_api = 'sncloud'
             else:  # SambaStudio
                 additional_env_vars = [
-                    'SAMBASTUDIO_BASE_URL',
-                    'SAMBASTUDIO_BASE_URI',
-                    'SAMBASTUDIO_PROJECT_ID',
-                    'SAMBASTUDIO_ENDPOINT_ID',
-                    'SAMBASTUDIO_API_KEY',
+                    'SAMBASTUDIO_URL',
```
Comment (snova-jorgep) [inline benchmarking/streamlit/app.py:94]: SAMBASTUDIO_API_KEY is not also an extra required env var?

Comment (snova-kwasia) [reply benchmarking/streamlit/app.py:94]: @snova-jorgep @snova-rodrigom is there a resolution to this? Thanks

Comment (snova-rodrigom) [reply benchmarking/streamlit/app.py:94]: hey @snova-kwasia , yes we've talked about this and I'm working on it. I'll let you know once it's ready

Comment (snova-rodrigom) [reply benchmarking/streamlit/app.py]: Alright, `env_input_fields` has been changed in order to be integrated to this kit. This is not affecting other dependent kits, I've tested EKR and Search Assistant to make sure about it. If you're good with it we can resolve this comment, thanks guys!

**Review by snova-kwasia — APPROVED** [review]
LGTM @snova-rodrigom

**Pull request merged**